MetaMask/metamask-mobile
# chore: sync icon library with figma icon library

## **Description**
Sync Icon's asset folder with Icons from Figma
https://www.figma.com/file/HKpPKij9V3TpsyMV1TpV7C/DS-Components?type=design&node-id=401%3A2122&mode=design&t=xMFUABEaqPDkcTTZ-1
Added
- Home-2
- Plant
- Plus-and-minus
- Snaps-round
- Sparkle
- Tint

## **Related issues**

Fixes:

## **Manual testing steps**

1. Access storybook via changing the pointer in the index file
2. Go to the Icon story
3. Check all the new icons

## **Screenshots/Recordings**



### **Before**
N/A

### **After**
https://github.com/MetaMask/metamask-mobile/assets/14355083/416a9609-0dd9-4773-abef-a25baeab5b07

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
